src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -79,10 +79,10 @@ ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
                 shape_b_aligned.insert(shape_b_aligned.begin(), 1);
             }
 
-            if (matmul->get_transpose_a()) {
+            if (matmul->get_transpose_a() && !is_type<op::v1::Transpose>(fc_input_a.get_node())) {
```

Comment:
Could you please explain why Transpose before matmul is a blocker for `get_transpose_a/b()` extraction?

Reply:
Also not relevant